app/pages/analyst/application-review.tsx

```diff
@@ -61,9 +57,23 @@ class ApplicationReview extends Component<Props, State> {
           }
         }
         applicationRevision(id: $applicationRevisionId) {
-          ...ApplicationDetailsContainer_applicationRevision
+          id
           overrideJustification
           isCurrentVersion
+          statusesSincePageLoad: applicationRevisionStatusesByApplicationIdAndVersionNumber(
+            last: 1
+          )
+            @connection(
+              key: "ApplicationReview_statusesSincePageLoad"
+              filters: []
+            ) {
+            edges {
+              node {
+                applicationRevisionStatus
+              }
+            }
+          }
```

Comment:
I had to change the fetching of `applicationRevisionStatus` to a connection in order to be able to refresh the status data after making a decision.

The status is changed by creating a new `applicationRevisionStatus` entity. The `application_revision_status` computed column on `application_revision` returns the "last" status for that revision based on the highest id number, which is okay for a non-interactive page that only needs to load the data once. The old dropdown component faked a refresh using state, which I'd rather not do since there is quite a bit of rendering on this page based on the current status.

I don't love the loosy goosey `last: 1` query condition, but that mirrors what's returned by the existing computed column used for the dropdown. I suppose I could add a filter to `orderBy: rowId` for greater certainty 🤔